std/uni.d

```diff
@@ -1760,17 +1771,34 @@ alias sharSwitchLowerBound = sharMethod!switchUniformLowerBound;
     static void append(T, V)(ref T[] arr, V value)
         if (!isInputRange!V)
     {
+        if (arr.length == size_t.max) assert(0);
```

Comment:
Yes, sorry - but this is still dead code as there is no range that matches `if (!isInputRange!V)` :/